javascript/javascript_and_the_backend/using_baas_for_your_backend.md

```diff
@@ -21,7 +21,9 @@ You can learn how to build a back-end later using [Node.js](https://www.theodinp
 <div class="lesson-content__panel" markdown="1">
 
 1. [Explore the different services offered by Firebase](https://firebase.google.com/products). Pay particular attention to [Cloud Firestore](https://firebase.google.com/products/firestore) for now. You may or may not also want to use [Firebase Hosting](https://firebase.google.com/products/hosting) and/or any of the other services offered by Firebase, particularly [Cloud Storage](https://firebase.google.com/products/storage) and [Authentication](https://firebase.google.com/products/auth), but you probably won't need any of those for your first Firebase project.
-
+<div class="lesson-note lesson-note--warning">
```

Comment:
![Screenshot 2023-07-06 at 11 22 58](https://github.com/TheOdinProject/curriculum/assets/81025586/bd92aaf8-7dd0-4846-9d31-d18e32cd7372)

This does not look good in the lesson preview for me

Reply:
Somehow managed to turn `</p>` into `<p>` between previewing and committing; should be good now